pxr/usdImaging/usdImagingGL/testenv/testUsdImagingGLPickAndHighlight.cpp

```diff
@@ -238,6 +298,17 @@ My_TestGLDrawing::DrawTest(bool offscreen)
     } else {
         Draw();
     }
+
```

Comment:
Missed this before, but can you add an if (renderer == HdStormRendererPlugin) here? Or else add a command line flag to control whether we test deep selection.